Classes/Common/AbstractDocument.php

```diff
@@ -92,7 +92,7 @@ abstract class AbstractDocument
     protected array $formats = [
         'OAI' => [
             'rootElement' => 'OAI-PMH',
-            'namespaceURI' => 'http://www.openarchives.org/OAI/2.0/',
+            'namespaceURI' => 'https://www.openarchives.org/OAI/2.0/',
```

Comment:
This is a XML namespace and should not be changed!
```suggestion
            'namespaceURI' => 'http://www.openarchives.org/OAI/2.0/',
```